### PR TITLE
Fix type guesser. Ignore field MappingException.

### PR DESCRIPTION
**Pull request by BoShurik**
Fix errors like "No mapping found for field 'plainPassword' in class 'App\Document\Administrator'."

### CODE REVIEW

File: Form/DoctrineMongoDBTypeGuesser.php [hunk 1]
```diff
@@ -49,6 +49,10 @@ public function guessType($class, $property)
 
         list($metadata, $name) = $ret;
 
+        if (!$metadata->hasField($property)) {
```
[the remark this answers is not in the document]
Comment (BoShurik) [reply Form/DoctrineMongoDBTypeGuesser.php]: @malarzm Done. But in code there are only 3 places with such style. And 22 without spaces

[the remark this answers is not in the document]
Comment (malarzm) [reply Form/DoctrineMongoDBTypeGuesser.php]: Ah so the bundle did not receive CS fixes yet, thank you :)

File: Form/DoctrineMongoDBTypeGuesser.php [hunk 2]
```diff
@@ -49,6 +49,10 @@ public function guessType($class, $property)
 
         list($metadata, $name) = $ret;
 
+        if (! $metadata->hasField($property)) {
+            return new TypeGuess($this->typeFQCN ? TextType::class : 'text', [], Guess::LOW_CONFIDENCE);
```
Comment (malarzm) [inline Form/DoctrineMongoDBTypeGuesser.php]: Now I saw the test I'm wondering whether guessing `TextType` (even with low confidence) is correct. From what I saw in https://symfony.com/doc/current/form/type_guesser.html it's possible to just `return;` if we can't (or shouldn't) make a guess at all. Do you think it'd be more appropiate?

Comment (BoShurik) [reply Form/DoctrineMongoDBTypeGuesser.php]: I copy behaviour from a [symfony/doctrine-bridge](https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php#L88). Looks like in the first versions type `guessType` returns only `TypeGuess` instance (based on phpDoc). I think now we can return nulls

Comment (malarzm) [reply Form/DoctrineMongoDBTypeGuesser.php]: Also if you could squash commits later that'd be awesome :)

Comment (BoShurik) [reply Form/DoctrineMongoDBTypeGuesser.php]: Done

**Comment by BoShurik** [comment]
@malarzm ping! Is there any other things that prevent merging?

**Comment by malarzm** [comment]
Sorry, had this email starred but did not get back to it in time. Looks good to me! @alcaeus are we couting this as a bugfix and releasing 3.4.3 or will this wait for 3.5.0?

**Comment by malarzm** [comment]
FWIW I'm fine with relasing 3.4.3 with this fix only as we do not get that many issues to fix.

**Comment by alcaeus** [comment]
Bugfix it is.

**Pull request merged**